salt/modules/pip.py

```diff
@@ -612,6 +614,24 @@ def install(pkgs=None,  # pylint: disable=R0912,R0913,R0914
     no_cache_dir
         Disable the cache.
 
+    extra_args
+        pip keyword and positional arguments not yet implemented in salt
```

Comment:
`pip keyword and positional arguments to be passed as parameters to the pip command`

I don't think we need to denote `not yet implemented`